step-templates/f5-ltm-rest-enable-disable-member.json

```diff
@@ -0,0 +1,114 @@
+  },
+  "Parameters": [
+    {
+      "Id": "5ed914ec-ff22-436d-ac37-1ccc5216372e",
+      "Name": "WaitForConnections",
```

Comment:
Can you prefix the variable names with the step template name? `F5LTM.WaitForConnections` for example.

Reply:
prefixed all variables with F5LTM in commit 75485dceb0a730c5a59929c40ef775ff7aaa5006